examples/google-adk/README.md

```diff
@@ -0,0 +1,9 @@
+# Gemini API Google ADK Examples
+
+## Table of contents
+
+This folder contains examples demonstrating how to use the Google Application Development Kit (ADK) with the Gemini API to build more robust, stateful, and manageable applications.
```

Comment:
You should add a link to the ADK's github project, and maybe copy-paste their intro to explain what it is..